### ISSUE TITLE
Two or more instances?

### ISSUE THREAD

**Issue by seros00** [davecoffin/nativescript-filterable-listpicker#58]
Can I have two or more instances in the same project?

**Comment by davecoffin** [comment]
Yes, but the more typical use case is to have one at the root and change the items you pass to it.

**Comment by seros00** [comment]
So, two instances can not be independent, i.e., each one with their own items?

**Comment by davecoffin** [comment]
Yes, but i just dont see a good reason to do that. Are they appearing on the screen at the same time?

**Comment by seros00** [comment]
Yes. Indeed I am using tabs and I need one list in one tab, and two (or more) in other tab, all of the lists appearing at the same time.
